src/client.js

```diff
@@ -469,8 +470,15 @@ export class Saturn {
   reportLogs (log) {
     this.logs.push(log)
     if (!this.reportingLogs) return
-    this.reportLogsTimeout && clearTimeout(this.reportLogsTimeout)
-    this.reportLogsTimeout = setTimeout(this._reportLogs.bind(this), 3_000)
+    clearTimeout(this.reportLogsTimeout)
```

Comment:
```suggestion
    this.reportLogsTimeout && clearTimeout(this.reportLogsTimeout)
```

this.reportLogsTimeout may be null